src/dialogs/examples/basic/O365ConnectorCardActionsDialogV1.ts

```diff
@@ -0,0 +1,206 @@
+import * as builder from "botbuilder";
+import { TriggerActionDialog } from "../../../utils/TriggerActionDialog";
+import { DialogIds } from "../../../utils/DialogIds";
+import { DialogMatches } from "../../../utils/DialogMatches";
+import { Strings } from "../../../locale/locale";
+import * as teams from "botbuilder-teams";
+
+export class O365ConnectorCardActionsDialogV1 extends TriggerActionDialog {
+
+    private static async step1(session: builder.Session, args?: any | builder.IDialogResult<any>, next?: (args?: builder.IDialogResult<any>) => void): Promise<void> {
+        // multiple choice examples
+        let cardAction1 = new teams.O365ConnectorCardActionCard(session)
+                            .id("CardsTypeSection1")
```

Comment:
Nit: please fix the indentation in this file.  For example, this line seems randomly aligned.

Reply:
can you please explain bit more about the indent issue, no tslint error.

Reply:
Typically we'd indent 4 spaces relative to the previous line. Line 13 (for example) is indented much more.

Reply:
Done.